### ISSUE TITLE
Unexpected sentence break when parentheses immediately follow abbreviation with period

### ISSUE THREAD

**Issue by reczy** [diasks2/pragmatic_segmenter#46]
Hi Kevin - first of all, thanks for your work on this gem.

I'd like to report the following unexpected behavior:

### Example 1: Unexpected Result:
Note the period in Inc.
```
PragmaticSegmenter::Segmenter.new(text: 'The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc. (“Company A”), and PragmaticSegmenterExampleCompanyB Inc. (“Company B”).', clean: false).segment
["The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc.", "(“Company A”), and PragmaticSegmenterExampleCompanyB Inc.", "(“Company B”)."]
```

### Example 2: Expected Result:
No period in Inc
```
PragmaticSegmenter::Segmenter.new(text: 'The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc (“Company A”), and PragmaticSegmenterExampleCompanyB Inc (“Company B”).', clean: false).segment
["The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc (“Company A”), and PragmaticSegmenterExampleCompanyB Inc (“Company B”)."]
```

### Example 3: Expected Result:
Note period in Inc. but now there's text between Inc. and the parens
```
PragmaticSegmenter::Segmenter.new(text: 'The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc., a fake corporation (“Company A”), and PragmaticSegmenterExampleCompanyB Inc., a fake corporation (“Company B”).', clean: false).segment
["The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc., a fake corporation (“Company A”), and PragmaticSegmenterExampleCompanyB Inc., a fake corporation (“Company B”)."]
```

### Example 4: Expected Result:
Same as Example 3 but without the comma after Inc.
```
PragmaticSegmenter::Segmenter.new(text: 'The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc. a fake corporation (“Company A”), and PragmaticSegmenterExampleCompanyB Inc. a fake corporation (“Company B”).', clean: false).segment
["The parties to this Agreement are PragmaticSegmenterExampleCompanyA Inc. a fake corporation (“Company A”), and PragmaticSegmenterExampleCompanyB Inc. a fake corporation (“Company B”)."]
```

**Comment by diasks2** [comment]
@reczy Thanks for reporting! I'll add those failing cases as specs and look into the issue.

**Comment by diasks2** [comment]
@reczy this should now be fixed in the latest version: 0.3.19. Let me know if you still experience any issues.

**Comment by reczy** [comment]
Looks like it's working for me as well! Thank you @diasks2
